enforce-rules.js

```diff
@@ -376,5 +376,5 @@ function allowMessage(message, humanBudget, queue, responseTimeLimit, canTalkAtO
 
 // Export these functions
 exports = module.exports = {
-  allowMessage, rule1Evaluation, rule4Evaluation
+  rule1Evaluation, rule4Evaluation, isSpeakerBot, rule0Evaluation
```

Comment:
For this, let's do this:
```suggestion
  rule0Evaluation,
  rule1Evaluation,
  rule4Evaluation,
  isSpeakerBot,
```

and then for each new exported function, add it as a new row with a trailing comma (e.g. `rule2Evaluation,`)